c7n/filters/securityhub.py

```diff
@@ -0,0 +1,79 @@
+# Copyright 2017 Capital One Services, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from c7n.utils import local_session, chunks, type_schema
+from .core import Filter
+from c7n.manager import resources
+
+
+class SecurityHubFindingFilter(Filter):
+    """Check if there are Security Hub Findings related to the resources
+    """
+    schema = type_schema('findings')
```

Comment:
ideally, there should be a lot of filter options available for a user here that can pass through.

Reply:
Agreed, captured in line:                 # TODO: Support parameterized filters rather not just finding exists

Reply:
I think we'll need this out of the box within this pr, simply filtering to resources with any finding, doesn't seem per se useful without also being to narrow on specific types of findings.

Reply:
Agreed, I was trying to indicate that in my PR description.